agnocast_kmod/agnocast_kunit/agnocast_kunit_increment_rc.c

```diff
@@ -4,14 +4,66 @@
 
 #include <kunit/test.h>
 
-// Feel free to delete this test case
-void test_case_increment_rc_sample0(struct kunit * test)
+static char * topic_name = "/kunit_test_topic";
```

Comment:
Please use `const` for const variables, and use CAPITAL_CASE for global const variables :+1: